packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/method_channel/method_channel_google_maps_flutter.dart

```diff
@@ -167,6 +167,12 @@ class MethodChannelGoogleMapsFlutter extends GoogleMapsFlutterPlatform {
     return _events(mapId).whereType<MapLongPressEvent>();
   }
 
+  @override
+  Stream<MapActiveLevelChangedEvent> onActiveLevelChanged(
```

Comment:
The changes to this file should be reverted; the method channel code is only for compatibility with any legacy third-party implementations, and by definition legacy implementations won't have support for new features.